.changeset/gold-moles-punch.md

```diff
@@ -0,0 +1,7 @@
+---
+"@latticexyz/world-module-erc20": patch
```

Comment:
since we haven't released this module yet, we can update the previous changeset instead of a new one!

Reply:
oh right, makes sense, ty